### PR TITLE
[TF-TRT] Fix SUPER call in vgg_block_test.

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**
Small fix in one of TF-TRT test file

### CODE REVIEW

**Comment by DEKHTIARJonathan** [comment]
@bixia1 for review

File: tensorflow/python/compiler/tensorrt/test/vgg_block_test.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def ExpectedEnginesToBuild(self, run_params):
 
   # TODO(b/159459919): remove this routine to disallow native segment execution.
   def setUp(self):
-    super(trt_test.TfTrtIntegrationTestBase, self).setUp()  # pylint: disable=bad-super-call
+    super(VGGBlockTest, self).setUp()  # pylint: disable=bad-super-call
```
Comment (pjannaty) [inline tensorflow/python/compiler/tensorrt/test/vgg_block_test.py:63]: Shouldn't this be `super().setUp()` according to [pep-3135](https://peps.python.org/pep-3135/)?

Comment (bixia1) [reply tensorflow/python/compiler/tensorrt/test/vgg_block_test.py:63]: Please also remove # pylint.

Comment (DEKHTIARJonathan) [reply tensorflow/python/compiler/tensorrt/test/vgg_block_test.py]: @bixia1 fixed
closing

**Comment by DEKHTIARJonathan** [comment]
@bixia1 removed all pylint commands and fix other wrong `super(Baseclass, self)....`calls I found in the codebase.

Please review

**Comment by bixia1** [comment]
We can actually remove the class name, see [here](https://github.com/tensorflow/tensorflow/blob/cf3093b448aca47faee00dac2fbb08f0e52b741a/tensorflow/python/compiler/tensorrt/test/shape_output_test.py#L79) in the same test suite.

**Comment by pjannaty** [comment]
> We can actually remove the class name, see [here](https://github.com/tensorflow/tensorflow/blob/cf3093b448aca47faee00dac2fbb08f0e52b741a/tensorflow/python/compiler/tensorrt/test/shape_output_test.py#L79) in the same test suite.

Yeap, I had pointed that out [here](https://github.com/tensorflow/tensorflow/pull/56486#discussion_r910281625), but I think Jonathan might have missed it.

**Review by bixia1 — APPROVED** [review]

**Pull request merged**